src/els_parser.erl

```diff
@@ -216,7 +216,7 @@ attribute(Tree) ->
           []
       end;
     {record, {Record, _Fields}} ->
-      [els_poi:new(Pos, record, atom_to_list(Record))];
+      [els_poi:new(Pos, record, Record)];
```

Comment:
The three changes to the parser ensure that both records and macros are handled consistently, allowing to re-use the same piece of code for code completion.